client/main.go

```diff
@@ -5,7 +5,9 @@ import (
 	"crypto/x509"
 	"io/ioutil"
 	"net/http"
+	// "math/big"
 	"net/url"
+	// "fmt"
```

Comment:
Please don't comment out imports. Golang plugin for VSC should automatically remove redundant imports.

Reply:
Kk